README.md

```diff
@@ -149,5 +161,16 @@ make check
 
 ### Project Status (Archive)
 
+#### 2018-08-07
+libiocage is making small but continuous steps to stabilize the interfaces and become used in [iocage/iocage](https://github.com/iocage/iocage).
+The project was first presented in the talk "[Imprisoning software with libiocage](https://www.bsdcan.org/2018/schedule/events/957.en.html)" at BSDCan 2018 (Video Recording on [YouTube](https://www.youtube.com/watch?v=CTGc3zYToh0)).
+There will be a Tutorial about [Advanced container management with libiocage](https://2018.eurobsdcon.org/tutorial-speakers/#StefanGronke) on September 20th, 2018 at EuroBSDCon in Bucharest.
+
+Ongoing preparations at this repository and iocage ensure that the transition to using libiocage under the hood of iocage go as smooth as possible for users.
+Features that exist in iocage will be further improved and tested or announced to be replaced or deprecated shortly.
+iXsystems let one imagine that libiocage once finds its way into FreeNAS where it can play its full strength behind a Web GUI.
```

Comment:
what does this sentence mean?

Reply:
This is archived news. It means that iXsystems is considering to use libiocage under the hood of FreeNAS. Work in progress.